### PR TITLE
[#164895584] Fixed search input width

### PR DESCRIPTION
**Pull request by Alex193a**
Fixed the search input width: it now uses all the available space.

### CODE REVIEW

File: ts/screens/messages/MessagesHomeScreen.tsx [hunk 1]
```diff
@@ -119,7 +119,7 @@ class MessagesHomeScreen extends React.Component<Props, State> {
         hideHeader={searchText.isSome()}
         headerBody={
           searchText.isSome() ? (
-            <Item>
+            <Item style={{ width: "150%" }}>
```
[the remark this answers is not in the document]
Comment (Alex193a) [reply ts/screens/messages/MessagesHomeScreen.tsx]: I'm trying it right now.

File: ts/components/screens/BaseHeader.tsx [hunk 2]
```diff
@@ -33,18 +33,18 @@ export class BaseHeader extends React.PureComponent<Props> {
     return (
       <AppHeader primary={this.props.primary} noLeft={goBack !== true}>
         {goBack && (
-          <Left>
+          <Left style={{ flex: 0 }}>
```
Comment (fpersico) [inline ts/components/screens/BaseHeader.tsx]: The change must be done in the theme of the native-base Header component.

Comment (fpersico) [reply ts/components/screens/BaseHeader.tsx]: https://github.com/teamdigitale/io-app/blob/master/ts/theme/components/Header.ts

File: ts/theme/components/Header.ts [hunk 3]
```diff
@@ -26,8 +26,8 @@ export default (): Theme => {
         height: (variables.iconSizeBase * 5) / 3
       },
       paddingLeft: 0,
-      flex: 0,
-      minWidth: 40
+      minWidth: 40,
```
Comment (fpersico) [inline ts/theme/components/Header.ts]: Just a lines reorder, not needed.

**Pull request merged**